lib/testmap.py

```diff
@@ -164,10 +164,10 @@ def contexts(image: str, *scenarios: Iterable[str], repo: str | None = None) ->
             'rhel-8-10',
             'rhel-9-4',
         ],
-        # These can be triggered manually with bots/tests-trigger
```

Comment:
A bit odd to remove that comment, but I don't mind..

Reply:
figured it was a bit out of place ._.